quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -397,16 +424,41 @@ impl Limits {
     }
 }
 
+#[must_use]
+#[derive(Debug)]
+pub struct UpdatableLimits<'a>(&'a mut Limits);
+
+impl<'a> UpdatableLimits<'a> {
+    pub fn new(limits: &'a mut Limits) -> UpdatableLimits<'a> {
+        UpdatableLimits(limits)
+    }
+
+    pub fn stream_batch_size(&mut self, size: u8) {
```

Comment:
I would prefer to call these `set_`, just in case we need to add getters in the future.

```suggestion
    pub fn set_stream_batch_size(&mut self, size: u8) {
```